feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/SearchSessionsFragment.kt

```diff
@@ -157,6 +157,7 @@ class SearchSessionsFragment : DaggerFragment() {
             ContextCompat.getColor(requireContext(), R.color.search_close_icon)
         )
         searchView.isIconified = false
+        searchView.setQuery(searchSessionsViewModel.uiModel.value!!.searchResult.query, false)
```

Comment:
You can use uiModel.requireValue() for it 🙏

Reply:
Let me think some minutes 🙇

Reply:
Thank you for your PR. I thought a little based on your plan.
How about this?

```kotlin
data class UiModel(val searchResult: SearchResult) {
        fun isEmpty() = this == EMPTY

        companion object {
            val EMPTY = UiModel(SearchResult.EMPTY)
        }
    }
```

```
val searchResult = searchSessionsViewModel.uiModel.requireValue().searchResult
if (!searchResult.isEmpty()) {
  searchView.setQuery(searchResult.query, false)
}
```

Reply:
I checked about LiveData.requireValue(). I should have used that.
Thank you for teaching 🙏🙏

and...
thank you code sample😭 I'll repair PR now.